packages/ui/src/TapToEdit.tsx

```diff
@@ -97,12 +97,21 @@ export const TapToEdit = ({
   description: propsDescription,
   showDescriptionAsTooltip = false,
   onlyShowDescriptionWhileEditing = true,
+  instanceId,
```

Comment:
What are we planning to use as the instanceid from the parent component? Just the document id?

Reply:
instanceId is like react key. It can be any unique identifier. Will update comment in common

Reply:
alright sorry about the confusion this is all set to go now